nengo/utils/simulator.py

```diff
@@ -83,10 +83,6 @@ def validate_ops(sets, ups, incs):
     for node in sets:
         assert len(sets[node]) == 1, (node, sets[node])
 
-    # -- assert that only one op updates any particular view
-    for node in ups:
-        assert len(ups[node]) == 1, (node, ups[node])
```

Comment:
Yeah, I think this is something we need to consider... I'm sure the difference (in this case) is pretty small, but it could be large in some cases. I'm not sure how we'd order ops that both update the same signal... So yeah, I'd be keen to keep this assertion in and figure out another way to do these updates. I think the `PreserveValue` op from before was pretty good.